docs/pages/access-controls/idps/saml-guide.mdx

```diff
@@ -15,69 +15,53 @@ authenticate to external services.
 
 - (!docs/pages/includes/tctl.mdx!)
 - If you're new to SAML, consider reviewing our [SAML Identity Provider
-Reference](./saml-reference.mdx) before proceeding.
+  Reference](./saml-reference.mdx) before proceeding.
+- User with permission to create Service Provider resource. The preset `editor` role has this permission.
+- Test SAML application, also known as SAML Service Provider (SP). For this guide, we'll be using
+[samltest.id](https://samltest.id/) to create a test consumer of Teleport's SAML IdP.
 
-## Example external application
+## Step 1/3. Add samltest.id Service Provider to Teleport
 
-We'll be using [samltest.id](https://samltest.id/) to create a test consumer of
-Teleport's SAML identity provider.
-
-## Step 1/4. Configure a Teleport role with access to SAML service provider objects
-
-First, ensure you are logged into Teleport as a user that has permissions
-to read and modify `saml_idp_service_provider` objects. The default `editor` role
-has access to this already, but in case you are using a more customized configuration,
-create a role called `sp-manager.yaml` with the following contents:
+The minimum configuration values required to add a Service Provider are:
+1. **Entity ID:** The SAML metadata endpoint of the Service Provider.
+2. **ACS URL:** The endpoint where users will be redirected after SAML authentication. ACS URL
+is also referred to as SAML SSO URL.
 
+The following `saml_idp_service_provider` spec is a reference for adding samltest.id to Teleport:
 ```yaml
-kind: role
+kind: saml_idp_service_provider
 metadata:
-  name: sp-manager
+  # The friendly name of the service provider. This is used to manage the
```

Comment:
We should capitalize "Service Provider" consistently. Using lowercase everywhere probably makes more sense, since this is a general computing term rather than a Teleport product.